docs/zh/reference/sql/dql/GROUP_BY_CLAUSE.md

```diff
@@ -16,26 +14,23 @@ SELECT select_expr [,select_expr...] FROM ... GROUP BY ...
 ```
 
 ## 边界说明
+在单机版中，所有执行模式均支持`GROUP BY`。集群版各执行模式的支持情况如下。
 
-| SELECT语句元素  | 状态          | 说明                                                         |
-| :-------------- | ------------- | :----------------------------------------------------------- |
-| GROUP BY Clause | Online 不支持 | Group By 子句用于对查询结果集进行分组。分组表达式列表仅支持简单列。 |
-
-
+| SELECT语句元素                              | 离线模式  | 在线预览模式 | 在线请求模式 | 说明                                                                                                                                                                                                                                                                          |
+|:----------------------------------------| --------- | ------------ |--------|:----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| GROUP BY Clause                         | **``✓``** |           |        | Group By 子句用于对查询结果集进行分组。分组表达式的列表仅支持直接给出列名，如`group by c1,c2,...` ，不支持较复杂的写法。 |
 
 ## Example
 
 ### 1. 按列分组后聚合
```

Comment:
pls help remove the section formating for those example titles, using bold is good enough

Reply:
the titles have been changed